Resources/Prototypes/_EinsteinEngines/Entities/Modsuits/security-responsory.yml

```
@@ -0,0 +1,160 @@
+            components:
+              - PowerCell
+    - type: Storage
+      maxItemSize: Huge
+      grid:
+        - 0,0,10,3
+    - type: ExplosionResistance #Dont want chain explosions in the modsuits storage
+      damageCoefficient: 0.1
+    - type: SealableClothingVisuals
+      visualLayers:
+        back:
+        - state: equipped-BACKPACK-sealed
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 8

(indentation)